tests/test_form.py

```diff
@@ -10,6 +10,7 @@
 class TestForm(unittest.TestCase):
 
     def test_normalize_url(self):
+        ''' Make sure that normalize_url method works.'''
```

Comment:
All tests are usually to make sure that a method works consistently. It's usually better to have a more explanatory docstring. 

For example, it could be 

```python
def test_normalize_url(self):
    '''Checks URL are correctly normalized.'''
```

and 

```python
def test_domain_name(self):
    '''Checks the domain name extraction.'''
```

Be careful about white spaces to. The first docstring starts with a space. After these changes I will approve. Thanks again.